lib/livebook_web/live/session_live.ex

```diff
@@ -55,11 +56,13 @@ defmodule LivebookWeb.SessionLive do
            platform: platform,
            data_view: data_to_view(data),
            autofocus_cell_id: autofocus_cell_id(data.notebook),
-           page_title: get_page_title(data.notebook.name)
+           page_title: get_page_title(data.notebook.name),
+           env_vars: Settings.fetch_env_vars()
          )
          |> assign_private(data: data)
          |> prune_outputs()
          |> prune_cell_sources()
+         |> put_env_vars()
```

Comment:
@aleDsz we don't want to do this in LV, because that would replicate the work for every connected user. Those kind of things should be done in `Session`, so that it happens once.